src/Hit/Formatting.hs

```diff
@@ -0,0 +1,34 @@
+module Hit.Formatting
+  ( padLeft
+  , padRight
+  , maxLenOn
+  )
+where
```

Comment:
Let's formatt it according to our style guide:

```haskell
module Hit.Formatting
       ( padLeft
       , padRight
       , maxLenOn
       ) where

Reply:
Sorry. Forgot to disable brittany on save.